.github/workflows/upgrade-python-requirements.yml

```diff
@@ -35,6 +35,7 @@ jobs:
       - name: make upgrade
         run: |
           cd $GITHUB_WORKSPACE
+          pip install -r requirements/base.txt
```

Comment:
maybe run `make requirements` instead

Reply:
Running any `make` command causes the same error because the MakeFile requires `Sphinx` to be already installed. Updating the command to only install `sphinx` instead.